File: Inf-121/POO/def2_inf121/d2_e2_Caja/E2_caja/Principal.cs
using System;
namespace E2_caja
{
	public class Principal
	{
		public static void Main(){
			Console.WriteLine("\n\n\tDatos Estudiante: Cristhian Andres Escobar Herrera");
			Console.WriteLine("\tCI: 6908109 L.P.");
			// a)
			Caja c1 = new Caja();
			Caja c2 = new Caja("Azul Oscuro","buen estado");
			Caja c3 = new Caja("Morado Oscuro","mal estado",5);
			Objeto ob1 = new Objeto();
			Objeto ob2 = new Objeto("Linterna","15");
			c1.mostrar();
			c2.mostrar();
			c3.mostrar();
			ob1.mostrar();
			ob2.mostrar();
			//b)
			c3--;
			c3.mostrar();
			//c)
			c1 = c3-c1;
			c1.mostrar();
			c3.mostrar();
			//d)
			Console.WriteLine("\n--x-----------------------------x--");
			c1 = c1 == ob1; 
			Console.WriteLine("\n--x-----------------------------x--");
			Console.WriteLine("\n\n");
			Console.ReadKey();
		}
	}
}

File: Inf-121/POO/def1_inf121/E3_Smaestros/src/Principal.java
public class Principal {
	public static void main(String[] args) {
		System.out.println("");
		System.out.println("\tDatos Estudiante: Cristhian Andres Escobar Herrera");
		System.out.println("\tCI: 6908109 L.P.");
		// a) Instanciar 2 Maestros de manera diferente y un objeto sindicato. 
		Maestros m1 = new Maestros();
		Maestros m2 = new Maestros(2500);
		Sindicato s1 = new Sindicato();
		
		m1.mostrarMae();
		m2.mostrarMae();
		
		s1.mostarSind();

		// b) Afiliar a ambos maestros al sindicato si es que no existen. 
		s1.maeAfiliados(m1);
		s1.maeAfiliados(m2);
		s1.mostarSind();

		// c) Mostrar el sueldo del maestro afiliado con el cargo de “Presidente”
		s1.sPresi(m1);
		s1.sPresi(m2);

	}

}	
		

		

		